lib/events.ts

```diff
@@ -96,14 +96,17 @@ export async function trackCommand(commandSignature: string) {
 				username,
 			});
 		});
-		await mixpanel.track(`[CLI] ${commandSignature}`, {
-			distinct_id: username,
-			version: packageJSON.version,
-			node: process.version,
-			arch: process.arch,
-			balenaUrl, // e.g. 'balena-cloud.com' or 'balena-staging.com'
-			platform: process.platform,
-		});
+		// Don't actually call mixpanel.track() while running test cases
+		if (!process.env.BALENA_CLI_TEST_TYPE) {
```

Comment:
In the other commit, I added `api.expectGetMixpanel({ optional: true });` to all test cases that execute CLI commands. Alas, this still wasn't enough to avoid the CI errors mentioned in the PR description. Mixpanel requests are special in that the CLI does not `await` for a reply, so my current theory is that a request initiated in a previous test case (that finishes quickly) may only be "seen" by nock during the execution of a subsequent test, such that the previous tests sees no mixpanel request (and succeeds because of `optional: true`), and the next test case sees two mixpanel requests (and fails because of `persist: false`). One way of getting tests to pass might be to write `api.expectGetMixpanel({ optional: true, persist: true });`, but this really means _"I don't care whether or not this request is made, or how many times",_ in which case, I thought, we might as well disable the requests during test execution. Hence coming up with the env var conditional. `BALENA_CLI_TEST_TYPE` is set to either `source` or `standalone` in the npm scripts in `package.json`.